extensions/amp-access/0.1/amp-access.js

```diff
@@ -333,6 +340,20 @@ export class AccessService {
   }
 
   /**
+   * @return {!Promise<string>}
+   * @private
+   */
+  getEncodedReaderId_() {
+    if (!this.encodedReaderIdPromise_) {
+      this.encodedReaderIdPromise_ = this.getReaderId_().then(readerId => {
+        dev.error(TAG, "bernie: readerId=", readerId);
```

Comment:
Or you can even use `console.log` before you submit. It won't allow you to submit with `console.log`, so it's easy to cleanup.